Model/Source/DefaultOptions.php

```diff
@@ -69,11 +69,12 @@ public function getDefault($option)
         $total = self::$order->getGrandTotal();
         $settings = self::$helper->getStandardConfig('options');
 
-        if (isset($settings[$option . '_active']) &&
-            $settings[$option . '_active'] == '1' &&
-            $total > (int)$settings[$option . '_from_price']
-        ) {
+        if ($settings[$option . '_active'] == '1') {
+            if ($total > (int)$settings[$option . '_from_price']) {
                 return true;
+            } else {
+                return false;
```

Comment:
De oude code werkt hetzelfde, behalve dat `isset($settings[$option . '_active'])` weg is. Bijvoorbeeld; deze else return false is niet nodig, hij komt vanzelf bij die andere false. Dan heb je genoeg aan 1 if (zoals het was)

Reply:
Done!